Polyhedron/doc/Polyhedron/CGAL/Polyhedron_3.h

```diff
@@ -1096,6 +1114,13 @@ class Polyhedron_3 {
   */
   Vertex_iterator vertices_end();
 
+  /*!
+  returns a range of iterators over vertices.
+  \note While the value type of `Vertex_iterator` is `Vertex`, the value type of
+        `All_vertex_handles::iterator` is `Vertex_handle`
+  */
+  Vertex_handles vertex_handles();
```

Comment:
The documentation says `All_vertex_handles` but the return type is `Vertex_handles`.